src/pyfaf/opsys/fedora.py

```diff
@@ -220,74 +218,68 @@ def save_ureport(self, db, db_report, ureport, packages, flush=False, count=1):
 
     def get_releases(self):
         result = {}
-        collections = self._pkgdb.get_collections()["collections"]
+        url = "https://pdc.fedoraproject.org/rest_api/v1/releases/"
+        url += "?page_size=-1"
```

Comment:
What means -1 ?

Reply:
https://github.com/product-definition-center/product-definition-center/issues/504 :)

Reply:
It means that all of the results are on one page and not 20 items per page. With releases it is safe as there are around 30 items. With components it is more complicated. There is 18000 of them and for 18000 is 900 requests which is sooo slow.  However I pull there only names, so it is 18000 * len(name) so it is just fine.